CODE_OF_CONDUCT.md

```diff
@@ -26,47 +26,54 @@ ambiente positivo per la nostra comunità:
 
 Esempi di comportamento inaccettabile:
 
-- L'uso di linguaggio o immagini sessualizzate e l'attenzione sessuale o avance indesiderate
-- Comportamenti da troll, commenti offensivi e attacchi personali o politici
-- Molestie in pubblico o in privato
-- Pubblicazione di informazioni private altrui, ad esempio un indirizzo postale o elettronico, senza autorizzazione esplicita
-- Altri comportamenti che potrebbero ragionevolmente essere considerati inappropriati in un contesto professionale
+- Commenti verbali offensivi relativi a sesso, orientamento sessuale, disabilità, aspetto fisico, corporatura, etnia, religione, immagini sessuali in spazi pubblici, intimidazioni intenzionali, stalking, inseguimenti, molestie fotografiche o registrazioni, interruzione prolungata di discorsi o altri eventi, contatto fisico inappropriato e attenzioni sessuali indesiderate;
+- L'uso di linguaggio inappropriato, condivisione di immagini sessualizzate, attenzioni sessuali o avance indesiderate;
+- Comportamenti da troll;
+- Molestie in pubblico o in privato;
+- Pubblicazione di informazioni private altrui, ad esempio un indirizzo postale o elettronico, senza autorizzazione esplicita;
+  -- Pubblicazione e/o condivisione, sotto qualsiasi forma, di contenuti che invogliano, contengono o generano attività illegali;
+- Pubblicazione e/o condivisione, sotto qualsiasi forma, di contenuti che violano espressamente diritti di copyright;
+- Condivisione di link e/o contenuti senza essersi prima appurati che siano sicuri e privi di minacce.
+- Altri comportamenti che potrebbero ragionevolmente essere considerati inappropriati in un contesto professionale, come a titolo esemplificativo ma non esaustivo, l’uso di linguaggio offensivo comprensivo di, ma non limitato a, religione o ideologia;
 
 ## Adempimento e Responsabilità
 
-Le persone alla guida della comunità sono responsabili del chiarimento
+Le persone alla guida della comunità (gli Ambassador) sono responsabili del chiarimento
 e dell'applicazione degli standard di comportamento accettabili e sono tenuti
 ad intraprendere azioni correttive appropriate ed eque in risposta a qualsiasi
 caso di comportamento inaccettabile, minaccioso, offensivo o dannoso.
 
-Le persone alla guida della comunità hanno il diritto e la responsabilità
-di rimuovere, modificare o rifiutare commenti, commit, codice, modifiche dei
+Le persone alla guida della comunità (gli Ambassador) hanno il diritto e la responsabilità
```

Comment:
Hai già detto sopra che sono le persone alla guida della comunità puoi utilizzare solo il termine Ambassador d'ora in poi

```suggestion
Gli Ambassador hanno il diritto e la responsabilità
```

Reply:
"Gli" non è molto gender-neutral, per quello la ripetizione

Reply:
Hai ragione, risolvo le conversazioni!